### PR TITLE
Fix documentation code samples

### PR DESCRIPTION
**Pull request by malukenho**
I've used https://github.com/codelicia/xulieta.
I'm not sure about the usage of the tool for this project but could be an idea.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ $proxyClass = $proxyGenerator->generateProxy(
     new ClassGenerator('ProxyClassName')
 );
 
-eval('<?php ' . $proxyClass->generate());
+eval('<? ' . $proxyClass->generate());
```
Comment (Ocramius) [inline CHANGELOG.md:211]: This should really not use short tags

Comment (malukenho) [reply CHANGELOG.md:211]: Yep. https://github.com/codelicia/xulieta/issues/19 I can open the `php` tag at the beginning of the code sample as a workaround, or just ignore this one as we don't have the tool here.

Comment (malukenho) [reply CHANGELOG.md:211]: Which one do you prefer?

Comment (Ocramius) [reply CHANGELOG.md:211]: I think the tool has a bug here: this is a string inside an `eval()` block, but xulieta caught it as a code delimiter

Comment (malukenho) [reply CHANGELOG.md]: Fixed by opening the php tag at the beginning of the code sample. I do have an issue open for that.

**Pull request merged**